### PR TITLE
3185 unify pipeline tag

### LINKED ISSUE #3185: ai_ prometheus metrics produced with inconsistent `pipeline` tag
It's maybe not a bug but an inconvenience resulting in multiple similar tags being produced. 
* `pipeline` value produced in 
[monitor.AIProcessingError(err.Error(), pipeline, ...](https://github.com/livepeer/go-livepeer/blob/ai-video/server/ai_http.go#L368) is equal to `text-to-image`, `image-to-image`, `upscale`, `image-to-video`, `audio-to-text`, `segment-anything-2` - lowercased with `-` as a separator.
* In [submitSegmentAnything2](https://github.com/livepeer/go-livepeer/blob/ai-video/server/ai_process.go#L635-L702) it's produced as `segment anything 2` (lower case, space separated).
* [`no orchestrators available...`](https://github.com/livepeer/go-livepeer/blob/ai-video/server/ai_process.go#L1058-L1098) errors are produced using `capName` resulting in `Text to image`, `Image to image`, `Image to video`, `Upscale`, `Audio to text`, `Segment anything 2` (First letter uppercase, space separated).

And because of that when trying to filter based on `pipeline` on Grafana we get "duplicates":
Screenshot 2024-09-27 at 15 52 02

We need some unification.

### PR DESCRIPTION
**Pull request by pwilczynskiclearcode**
**What does this pull request do? Explain your changes.**
Unifies the "pipeline" prometheus metric tag produced by `AIRequestError`

Fixes #3185

**Checklist:**

- [ ] Read the [contribution guide](./CONTRIBUTING.md)
- [ ] `make` runs successfully
- [ ] All tests in `./test.sh` pass
- [ ] README and other documentation updated
- [ ] [Pending changelog](./CHANGELOG_PENDING.md) updated

### CODE REVIEW

File: monitor/census.go [hunk 1]
```diff
@@ -1924,3 +1927,7 @@ func FastVerificationFailed(ctx context.Context, uri string, errtype int) {
 		clog.Errorf(ctx, "Error recording metrics err=%q", err)
 	}
 }
+
+func normilizePipelineTag(pipeline string) string {
```
Comment (leszko) [inline monitor/census.go]: Typo
```suggestion
func normalizePipelineTag(pipeline string) string {
```

**Review by leszko — APPROVED** [review]
LGTM.

@rickstaa should we just merge it or should we notify anyone that the label is changed?

**Review by rickstaa — APPROVED** [review]
@pwilczynskiclearcode Great catch! I've implemented your suggested change to the new AI remote worker metrics functions. One additional point to consider is whether we want to ensure consistent behavior in the [ai_process.go](https://github.com/livepeer/go-livepeer/blob/16de762f9376f85f115cef377c70502f8cdc8b4f/server/ai_process.go#L1119) file. Currently, we're inserting a capability string instead of a pipeline string, as we are in [census.go](https://github.com/livepeer/go-livepeer/blob/16de762f9376f85f115cef377c70502f8cdc8b4f/monitor/census.go#L1889). 

I'm okay with handling this discrepancy in the census package for now, but if we think it's better to also address this in the core codebase, we can include a change in a subsequent pull request. @leszko, @pwilczynskiclearcode — let me know what you think!

**Comment by rickstaa** [comment]
@pwilczynskiclearcode, @leszko we can also change `pipeline string` to `pipelineTag string` to make the behavoir more clear from outside the census package.

**Comment by leszko** [comment]
Added one commit https://github.com/livepeer/go-livepeer/pull/3191/commits/421aa2cb6eb5e14fd5d5ce1856bca6668963a79c

I think I like the suggestion from @rickstaa to have this conversion outside the census code. We should use pipeline naming consistently through the code.

@pwilczynskiclearcode If you like my change, feel free to merge (after all CI builds pass).

**Pull request merged**